applications/translation-and-localisation.md

```diff
@@ -8,15 +8,35 @@ description: Machine translation for translation and localisation
 
 Machine translation is a key technology in professional **translation and localisation** workflows.
 
+### Definitions
+
+#### Translation
+Translation is the transformation of a text or speech transcript in the source language into a text in the target language.
+
+#### Localisation
+Localisation is the transformation of an object (a document, a computer programme, a film, a book) from one cultural environment to another, whereby the object retains its most salient features and features that are not transferable to another culture are replaced, as far as possible, by counterparts that are comprehensible to the audience.  
+Localisation of a computer programme consists of the following steps:
+1. Identifying the usability to target audiences and the features that need to be replaced or adapted.
+2. Translating the content of user interface and user assistance (help).
```

Comment:
We can probably leave out "help".

Reply:
Is the expression "user assistance" commonly known?

Reply:
I think it's clear from the context, so I removed it.

Reply:
You are right, thank you!